### DISCUSSION TITLE
Is it possible to limit vehicle status in the Requirement endpoint?

### DISCUSSION THREAD

**Discussion by edwinvandenbelt** [openmobilityfoundation/mobility-data-specification#866] [Policy]
When using 2.0 /vehicles/status' endpoint, can we limit the result of it (only providing assets that are available, or not in use)? We can limit the number of fields (projection), but I haven't found something to limit the results (selection).

**Comment by schnuerle** [comment]
You can apply Policy rules per available states only. See this link and the examples.
https://github.com/openmobilityfoundation/mobility-data-specification/tree/main/policy#rules

But there is not a way yet using Policy Requirements to enumerate just the relevant vehicle states you want data for. This could be a feature update for this.
